jenkins-integration/jenkins-jobs/common/scripts/jenkins/pipeline.groovy

```diff
@@ -51,6 +61,36 @@ def get_server_era(pe_version) {
     }
 }
 
+def get_oss_server_era(oss_version) {
+    // TODO: eventually we will probably want to do something more sophisticated
+    //  here; currently only support 'latest'/'master'/'stable' OSS puppetserver,
+    //  and 'latest' agent
+    if (["latest", "master", "stable"].contains(oss_version)) {
+        return [type: "oss",
+                service_name: "puppetserver",
+                version: oss_version,
+                agent_version: "latest",
+                tk_auth: false,
```

Comment:
I'd like to do that, but since the OSS packages still disable it by default (right?), I figured that in order to get that working I'd also have to add a step somewhere that changed that configuration setting.  I'd like to do that (can ticket it if you like), but didn't feel like it was a good ROI for now.